web/src/services/tracker.ts

```diff
@@ -69,6 +69,7 @@ export function trackRecording(
     | 'record'
     | 'submit'
     | 'rerecord'
+    | 'discard-ongoing'
```

Comment:
I think this is sufficient distinct action from "rerecord" to warrant a separate tracking key, but should maybe go in a separate commit?